packages/storybook/cxl-ui/cxl-marketing-nav.data.json

```diff
@@ -436,60 +436,60 @@
       "children": [
         {
           "depth": 1,
-          "text": "Account details",
+          "text": "Go to My Account",
+          "component": "a",
+          "href": "http:\/\/localhost:8080\/my-account\/",
+          "id": 87,
+          "parent": 41
+        },
+        {
+          "component": "hr"
+        },
+        {
+          "depth": 1,
+          "text": "Subscriptions",
+          "component": "a",
+          "href": "http:\/\/localhost:8080\/my-account\/subscriptions\/",
           "id": 88,
-          "parent": 41,
-          "children": [
-            {
-              "depth": 2,
-              "text": "Account hello!",
-              "component": "a",
-              "href": "http:\/\/localhost:8080\/hello-world\/",
-              "id": 92,
-              "parent": 88
-            },
-            {
-              "depth": 2,
-              "text": "Addresses",
-              "component": "a",
-              "href": "http:\/\/localhost:8080\/edit-address\/",
-              "id": 89,
-              "parent": 88,
-              "children": [
-                {
-                  "depth": 3,
-                  "text": "<span data-post-status=\"publish\"><\/span>Hello world!",
-                  "component": "a",
-                  "href": "http:\/\/localhost:8080\/hello-world\/",
-                  "id": 93,
-                  "parent": 89
-                },
-                {
-                  "depth": 3,
-                  "text": "Downloads",
-                  "component": "a",
-                  "href": "http:\/\/localhost:8080\/downloads\/",
-                  "id": 90,
-                  "parent": 89
-                }
-              ]
-            },
-            {
-              "depth": 2,
-              "text": "<span data-post-status=\"publish\"><\/span>Hello world!",
-              "component": "a",
-              "href": "http:\/\/localhost:8080\/hello-world\/",
-              "id": 91,
-              "parent": 88
-            }
-          ]
+          "parent": 41
         },
         {
           "depth": 1,
           "text": "Orders",
           "component": "a",
-          "href": "http:\/\/localhost:8080\/orders\/",
-          "id": 87,
+          "href": "http:\/\/localhost:8080\/my-account\/orders\/",
+          "id": 89,
+          "parent": 41
+        },
+        {
+          "depth": 1,
+          "text": "Account details",
+          "component": "a",
+          "href": "http:\/\/localhost:8080\/my-account\/edit-account\/",
+          "id": 90,
+          "parent": 41
+        },
+        {
+          "component": "hr"
+        },
+        {
+          "depth": 1,
+          "text": "Skip Trial Period",
+          "description": "Gain unlimited instant access to all CXL features.",
```

Comment:
Can y'all get some realistic descriptions from growth / product team, and add to this data file? Need better answers to "What exactly are they planning to do with this feature?"

It's difficult to make tech design decisions with just a single random lorem ipsum-like example.

Reply:
@saas786 @anoblet can we get it today? This is blocking two PRs:
* https://github.com/conversionxl/institute-plugin/pull/216
* https://github.com/conversionxl/wp-theme-lib/pull/62

and important feature @kylebrodeur is waiting for.

Reply:
I asked @saas786 , and he said this PR should be good to go. @pawelkmpt do you see any changes that you think should be made?

Reply:
I do not, @lkraav has.

Reply:
Yes I see #224 too.

I only added Trial related menu item and also synced live menu for `My Account` for testing puproses.

CXL Marketing Nav certainly should reflect institute menu items, Lorem Ipsum or Demo items doesn't help much.
Will allow us to test live menu items whenever we change something in CXL Marketing Nav or related elements.

Rather than discovering issues on live website.